extension/js/common/platform/store/key-store.ts

```diff
@@ -25,6 +25,12 @@ export class KeyStore extends AbstractStore {
     });
   }
 
+  public static getFirst = async (acctEmail: string): Promise<KeyInfo> => {
```

Comment:
This returns `KeyInfo | undefined` - it's better to make it explicit so that someone doesn't get surprised when using it later.

Reply:
Actually, we could put `Assert.abortAndRenderErrorIfKeyinfoEmpty(keys[0]);` at the end - that will ensure that indeed the returned type is `KeyInfo`. Then anytime we call `Assert.abortAndRenderErrorIfKeyinfoEmpty` manually after calling `KeyStore.getFirst`, we could remove these lines.

Reply:
If it's good now, maybe rename the method to `getFirstOrRenderError`?

Reply:
Good suggestion, we can rename it for clarity :+1: